deploy/cf-deployment-operations/add-istio.yml

```diff
@@ -108,11 +110,9 @@
       query: "*"
 
 - type: replace
-  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/app_domains?
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/app_domains/-
   value:
-    - name: "apps.internal"
```

Comment:
If we remove this we won't have an internal apps domain anymore and I am pretty sure CI will go red. Was this intentional?

Reply:
See line 113: I'm changing a list replace to a list append.  the `apps.internal` is already there, added by the ops-file for [enable-service-discovery](https://github.com/cloudfoundry/cf-deployment/blob/master/operations/enable-service-discovery.yml)

Reply:
perfect, thanks for clarifying.